### DISCUSSION TITLE
Duplicate authors

### DISCUSSION THREAD

**Discussion by prolibre** [bookwyrm-social/bookwyrm#3466] [Q&A]
I'm in the process of cleaning up duplicate authors (e.g. those with the same isni or goodreads_key).

I notice that some authors are not linked to any book (no entry in the “_bookwyrm_book_authors_” table). They therefore appear to be orphans. Can I delete them without any repercussions ?

Have a nice day !

Anthony

**Comment by codeinabox** [comment]
This appears to be related to #2076.
